lib/matplotlib/backends/backend_gdk.py

```diff
@@ -36,9 +36,8 @@ def fn_name(): return sys._getframe(1).f_code.co_name
 _debug = False
 
 # Image formats that this backend supports - for FileChooser and print_figure()
-IMAGE_FORMAT  = ['eps', 'jpg', 'png', 'ps', 'svg'] + ['bmp'] # , 'raw', 'rgb']
-IMAGE_FORMAT.sort()
-IMAGE_FORMAT_DEFAULT  = 'png'
+IMAGE_FORMAT = sorted(['eps', 'jpg', 'png', 'ps', 'svg'] + ['bmp']) # , 'raw', 'rgb']
```

Comment:
Sure, but I'll keep the `sorted` which makes the intent clear and should be cheap.